### ISSUE TITLE
training to enhance the two people capability 

### ISSUE THREAD

**Issue by syusukexrj** [tencent-ailab/IP-Adapter#183]
I want to enhance the capability of generating two-people picture and faithfully restore everyone's face id, clothing, glasses, hat and etc. Any suggestion about data preparation? Is it possible to achieve this by ip-adapter?

**Comment by xiaohu2015** [comment]
for two person, I think you can use attention_mask https://github.com/cubiq/ComfyUI_IPAdapter_plus?tab=readme-ov-file#attention-masking

**Comment by whiterose199187** [comment]
Hello,

Is there a way to do this without using comfy ?
